examples/terraform/azure/output.tf

```diff
@@ -22,6 +22,21 @@ output "kubeone_api" {
   }
 }
 
+# Hack to ensure we get access to public ip in first attempt
+resource "time_sleep" "wait_30_seconds" {
+  depends_on = [azurerm_virtual_machine.control_plane]
+  create_duration = "30s"
+}
+
+data "azurerm_public_ip" "control_plane" {
+  depends_on = [
+    time_sleep.wait_30_seconds
+  ]
+  count = var.control_plane_vm_count
+  name                = "${var.cluster_name}-cp-${count.index}"
+  resource_group_name = azurerm_resource_group.rg.name
+}
```

Comment:
Is it possible to define those resources in `main.tf` or is it required to put them in `output.tf`?

Reply:
I can move the resources in main.tf. I kept them here as they hacks only specifically added for the ensuring right output. But I will move them.